tools/local-environment/tests/addChainCase.ts

```diff
@@ -211,6 +203,103 @@ async function sendTokensTriangular(
   }
 }
 
+async function sendTokensHubAndSpoke(
+  a: Network,
+  b: Network,
+  c: Network,
+  aActor: Key,
+  bActor: Key,
+  cActor: Key,
+  n: Nomad
+) {
+  const tokenFactory = getCustomToken();
+  const tokenOnA = await a.deployToken(
+    tokenFactory,
+    aActor.toAddress(),
+    "MyToken",
+    "MTK"
+  );
+
+  const token = {
+    domain: a.domain,
+    id: tokenOnA.address,
+  };
+
+  const ctx = n.getMultiprovider();
+
+  ctx.registerWalletSigner(a.name, aActor.toString());
+  ctx.registerWalletSigner(b.name, bActor.toString());
+
+  ctx.registerWalletSigner(c.name, cActor.toString());
+
+  // get 3 random amounts which will be bridged
+  const amount1 = randomTokens();
```

Comment:
nit: rename to `getRandomTokenAmount`

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/1bb3987d6787c6855a1247fa4300f75a805cc6cf